src/background/toolbarButton.ts

```diff
@@ -45,7 +45,19 @@ async function registerBadgeChanges() {
   listenToStorageChange(
     'notificationsData',
     ({ newValue: notificationsData }) => {
+      // let notificationsDataLoadingState = "loading";
+      // notificationsDataLoadingState = refreshData();
+      // try {
+      //   notificationsDataLoadingState = "loaded";
+      //   await updateNotificationsData();
+        // browser.browserAction.setBadgeBackgroundColor({colors.interactive.gray});
+        // } catch {
+          //   notificationsDataLoadingState = "error";
+        // }
       if (notificationsData) {
+        const color = colors.interactive.gray;
+        browser.browserAction.setBadgeBackgroundColor({color});
+        browser.browserAction.setIcon({path: './assests/img/spinner-solid.svg'});
```

Comment:
My expectation was not to change the extension icon on the toolbar completely, but use the `🗘` unicode character as a text for the badge.